GoogleCloudPlatform/vertex-ai-samples
# chore, fix(egen): follows new template, markdown updates and fixes

<br>
This PR includes the following updates to the `challenger_vs_blessed_deployment_method.ipynb` notebook:

- Change REGION to LOCATION
- Remove IS_TESTING condition at cleaning step
- Adds IS_COLAB in the service account setup
- Removes USER in the installation
- Moves the deletion of local files to the cleaning up section
- Add Colab Enterprise link and other updates from new template.
- Reorganizes the heading styles to match the steps.
- Removes unnecessary code highlights
- Removes future tense 
- Contracts words like "do not" --> "don't"
- Update K80 GPU with T4 GPU

<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>
